platform/android/java/lib/src/org/godotengine/godot/io/file/DataAccess.kt

```diff
@@ -170,14 +170,17 @@ internal abstract class DataAccess(private val filePath: String) {
 		}
 	}
 
-	fun write(buffer: ByteBuffer) {
+	fun write(buffer: ByteBuffer): Boolean {
 		try {
+			val totalBytes = buffer.position()
 			val writtenBytes = fileChannel.write(buffer)
 			if (writtenBytes > 0) {
 				endOfFile = false
 			}
+			return (totalBytes == writtenBytes)
```

Comment:
I feel like as long as any bytes are written, we should return `true` especially as an error would trigger an exception.

Reply:
Not writing all bytes is definitely a failure, but if it will trigger an exception (which seems to be the case for the file), there's no need for this check.